core/src/main/java/com/netease/arctic/utils/TablePropertyUtil.java

```diff
@@ -108,6 +108,15 @@ public static Map<String, String> getPartitionProperties(
         partitionPath);
   }
 
+  public static Map<String, String> getPartitionProperties(
```

Comment:
It seems like `getPartitionProperties(ArcticTable arcticTable, String partitionPath)` and `getPartitionProperties(UnkeyedTable unkeyedTable, String partitionPath)` are no longer needed. Can we delete them?